app/models/order.rb

```diff
@@ -89,6 +89,25 @@ def set_initial_state
     self.state ||= :draft
   end
 
+  def is_priority?
+    now = Time.now.in_time_zone
+    last_6pm = now.change(hour: 18, min: 0, sec: 0)
+    last_6pm -= 24.hours if now < last_6pm
```

Comment:
Another way for this.

`last_pm = last_6pm.advance(days: -1) if now < last_6pm`

not necessary to implement.